samples/app-auth/nodejs/README.md

```diff
@@ -29,6 +29,7 @@ This sample demonstrates authentication in Microsoft Teams apps.
     ```
 
 1. Register a bot with Azure Bot Service, following the instructions [here](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-quickstart-registration?view=azure-bot-service-3.0).
+- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
 1. While registering the bot, use `https://<your_ngrok_url>/api/messages` as the messaging endpoint.
```

Comment:
Why this is added as a point 1. ?
Also I can see that the next point is also starting with 1. 
![image](https://user-images.githubusercontent.com/59052547/147834857-f5c22723-ada1-4e90-ad25-a733757c5034.png)